persistence.py

```diff
@@ -27,7 +28,9 @@ def init(config: dict, db_url: Optional[str] = None) -> None:
     """
     _CONF.update(config)
     if not db_url:
-        if _CONF.get('dry_run', False):
+        if misc.args.backtesting:
+            db_url = 'sqlite:///' + os.path.join(tempfile.gettempdir(), 'tradesv2.backtesting.sqlite')
```

Comment:
this db only holds transient data you could also use `sqlite://` as db_url to create it in memory.